### PR TITLE
New standardized stylelint config

### PR DESCRIPTION
**Pull request by cosmocoder**
We setup both a standardized config that can be used throughout Jimdo, and also the standard tooling used by Stylelint.

### CODE REVIEW

File: .dependabot/config.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# https://dependabot.com/docs/config-file/
```
Comment (cheyer) [inline .dependabot/config.yml:1]: Do we stil need to setup something on the `dependabot` app itself?

File: .github/workflows/nodejs.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+name: CI
```
Comment (cheyer) [inline .github/workflows/nodejs.yml:1]: Do Github Actions work by only providing this config file, or do we have to do something else?

File: index.js [hunk 3]
```diff
@@ -1,89 +1,36 @@
...
-    "value-keyword-case": "lower",
-    "value-list-comma-newline-after": "always-multi-line",
-    "value-list-comma-space-after": "always-single-line",
-    "value-list-comma-space-before": "never",
-    "value-no-vendor-prefix": true
-  }
-}
+  extends: ['stylelint-config-standard', 'stylelint-config-css-modules', 'stylelint-config-rational-order'],
+  plugins: ['stylelint-scss'],
+  rules: {
```
Comment (cheyer) [inline index.js:6]: How did you decide on the rules?

File: README.md [hunk 4]
```diff
@@ -1,2 +1,59 @@
 # stylelint-config-jimdo
-Custom Styling Configuration for Jimdo
+
```
Comment (cheyer) [inline README.md:2]: Nice readme 👍

File: .dependabot/config.yml (hunk 1 above)
Comment (cosmocoder) [reply .dependabot/config.yml:1]: If we have a config file like this then we don't need to setup anything else

File: .github/workflows/nodejs.yml (hunk 2 above)
Comment (cosmocoder) [reply .github/workflows/nodejs.yml:1]: As far as I can see this file is enough. The actions worked for this PR

File: index.js (hunk 3 above)
Comment (cosmocoder) [reply index.js:6]: These rules are taken from the logo-tool, which in turn was decided upon after studying the rules in cms-frontend and what are the best options and approaches available out there.

**Pull request merged**